config/business_time.yml

```diff
@@ -0,0 +1,16 @@
+business_time:
+  beginning_of_workday: 9:00 am
+  end_of_workday: 5:00 pm
+  holidays:
+    - September 28, 2015
+    - September 29, 2015
+    - September 30, 2015
+    - December 24, 2015
+    - May 4, 2016
+    - May 5, 2016
```

Comment:
Correct! These are not sample or testing values. They are offsite days and the day before christmas (half day). I think it makes sense to exclude offsite days to encourage people to be more present, but open to suggestions. @melodykramer, what do you think?